qutebrowser/config/websettings.py

```diff
@@ -193,6 +205,15 @@ def _update_setting(self, setting: str, value: typing.Any) -> bool:
             return self.set_font_family(setting, value)
         elif setting == 'content.default_encoding':
             return self.set_default_text_encoding(value)
+        elif setting == 'unknown_url.scheme.policy':
+            # QtWebKit and QWebEngine < 5.11 doesn't provide interfaces
+            # for processing UnknownUrlSchemePolicy.
+            #
+            # AttributeError is expected for such cases.
+            try:
+                return self.set_unknown_url_scheme_policy(value)
+            except AttributeError:
+                pass
```

Comment:
Might be cleaner to override the `_update_setting` method in `WebEngineSettings` - something like this (untested):

```python3
def _update_setting(self, setting: str, value: typing.Any) -> bool:
    if setting == 'unknown_url.scheme.policy':
        return self.set_unknown_url_scheme_policy(value)
    return super()._update_setting(setting, value)
```

You don't need to take care to handle Qt < 5.10 compatibility if you set the `backend` key correctly in `configdata.yml` as mentioned above, at least if I'm not mistaken.